tests/integration/podman/move_test_scripts_to_bin.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# This script moves all shell script files (.sh) from the source directory to
```

Comment:
I think it would be easier if you just removed the .sh extension from all files from the script folder directly and kept them executable instead of running post-processing as you did here.